packages/cmf/src/selectors/toJS.js

```diff
@@ -0,0 +1,26 @@
+function toJS(data) {
+	if (data) {
+		if (data.toJS) {
+			return data.toJS();
+		}
+		throw new Error('the selector return a data which is not an immutable');
```

Comment:
Enhancement of the error message
'The selector ${function.name} return a data which is not an immutable, and thus should bnot be used with `toJS` selector decorator'

for this you would propably need throw a more generic error to be catched into getToJSMemoized so it can fill the function name